client-app/src/components/TeacherCreateQuizView/TeacherCreateQuizView.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import CreateQuestionObject from '../createQuestionObject/CreateQuestionObject';
+
+class TeacherCreateQuizView extends Component {
+	constructor(props){
+		super(props);
+		this.state = {
+			   selectValue: "10"
```

Comment:
What does 'selectValue' mean here?  Maybe a more descriptive name for the variable?

Reply:
i changed the variable name to something more descriptive